doc/source/ray-core/user-guide.rst

```diff
@@ -14,6 +14,7 @@ If you’re brand new to Ray, we recommend starting with the :ref:`walkthrough <
     actors
     objects
     scheduling/index.rst
+    tasks/using-ray-with-gpus
```

Comment:
How about putting this under the scheduling section? It seems to be a subtopic rather than category of its own.

Reply:
Sure.